### PR TITLE
IST-14 Add eslint to project

### PR DESCRIPTION
**Pull request by Hemanth694**
## Description

Added Eslint to the project

## Issue link

https://internalsurveytool.atlassian.net/browse/IST-14

### CODE REVIEW

File: front-end/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
```
Comment (Manikumar1998) [inline front-end/.eslintrc.js]: This should be a typescript file not JavaScript. Find a typescript example.

File: front-end/src/App.test.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import Header from './Header';
+import UserHomePage from './UserHomePage';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  render(<Header />);
+  const linkElement = screen.getByText(/welcome back hemanth/i);
```
Comment (Manikumar1998) [inline front-end/src/App.test.tsx]: Does this test even pass? ‘Welcome back Hemanth’ is rendered in the UserHomepage.tsx file not in Header

File: front-end/src/Header.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import TopNavigation from "@cloudscape-design/components/top-navigation";
 
 export default function Header(){
     return(
+        // eslint-disable-next-line react/react-in-jsx-scope
```
Comment (Manikumar1998) [inline front-end/src/Header.tsx]: Why are we disabling it? We should fix the es lint issue instead

File: front-end/src/App.test.tsx (hunk 2 above)
Comment (Hemanth694) [reply front-end/src/App.test.tsx]: It doesn't, Like I've mentioned before I'm still working on it.

File: front-end/src/App.tsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/react-in-jsx-scope */
```
Comment (Manikumar1998) [inline front-end/src/App.tsx]: Same here. Why are we disabling it?

Comment (Hemanth694) [reply front-end/src/App.tsx]: I used to get an error that goes "React out of scope", I disabled it temporarily and now I've removed it from the files and it works fine. 

I'm going to make another commit right now without disabling any eslint configurations but please keep in mind that the app.test.tsx is still on development.

**Review by Manikumar1998 — APPROVED** [review]

**Pull request merged**